docs/developer-guide/mapstore-migration-guide.md

```diff
@@ -22,6 +22,44 @@ This is a list of things to check if you want to update from a previous version
 
 ## Migration from 2023.02.xx to 2024.01.00
 
+### Adding spatial filter to dashboard widgets
+
+In order to enable the possibility to add in and the spatial filter to the widgets [#9098](https://github.com/geosolutions-it/MapStore2/issues/9098) you have to edit the QueryPanel config in `localConfig.json` file adding:
+
+- **useEmbeddedMap** flag,
+- **spatialOperations**
+- **spatialMethodOptions**
+
+  *Note: these props can vary, for example you can limit the spatial filtering operation to intersect only*
+
+```json
+...
+dashboard: [
+...
+{ 
+    "name": "QueryPanel",
+    "cfg": {
+        "toolsOptions": {
+            "hideCrossLayer": true,
+            "useEmbeddedMap": true
+        },
+        "spatialPanelExpanded": false,
+        "spatialOperations": [
+            {"id": "INTERSECTS", "name": "queryform.spatialfilter.operations.intersects"},
+            {"id": "CONTAINS", "name": "queryform.spatialfilter.operations.contains"},
+            {"id": "WITHIN", "name": "queryform.spatialfilter.operations.within"}
+        ],
+        "spatialMethodOptions": [
+            {"id": "BBOX", "name": "queryform.spatialfilter.methods.box"},
+            {"id": "Circle", "name": "queryform.spatialfilter.methods.circle"},
+            {"id": "Polygon", "name": "queryform.spatialfilter.methods.poly"}
+        ],
```

Comment:
Is it possible to avoid one of these? I know that here viewport will be removed, but at least we can use the defaults for spatialOperations?

Reply:
this would mean to alter the current default props? do you want this?
without the spatial operations you get this now
![image](https://github.com/geosolutions-it/MapStore2/assets/11991428/929c4df6-10d3-4136-a15c-dc644c269eef)

Reply:
Ah, ok. So the defaults are very old and probably wrong, and they should be updatd. Spatial Operations too?

Reply:
I tested it personally. Without specifying it, the tool do not work. This is a limitation of the query panel, but not part of this PR. 
Maybe we will make the query panel have a better default configuration, without needing to specify